packages/layouts-css/src/page.scss

```diff
@@ -94,5 +96,71 @@
         justify-content: flex-end;
       }
     }
+
+    &-centered {
+      max-width: 1440px;
+      width: 100%;
+    }
+
+    &-title-area {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: calc(var(--iui-size-s) * 3);
+    }
   }
+
+  &-bottom-bar {
+    grid-area: bottom-bar;
+    height: calc(var(--iui-size-s)*5 + var(--iui-size-l)); //this is the sizes of the two bottom buttons of side nav
+  }
+
+  &-action-bar {
+    background-color: var(--iui-color-background);
+    border-top: 2px solid var(--iui-color-border);
+    margin-inline: auto;
+    padding: var(--iui-size-m);
+    height: inherit;
+    display: grid;
```

Comment:
Add small `gap` to create separation between areas if users add a lot of things into them

Reply:
Added gap with size `--iui-size-xs`